### PR TITLE
fix(helptext): update slot name

### LINKED ISSUE #75: Slot name inconsistent with other elements
The following slot name is inconsistent with other slot names of similar use 

https://github.com/AlaskaAirlines/auro-dropdown/blob/e1a24170e0c98ff82ccce581a90423e9fe94754c/src/auro-dropdown.js#L26

From auro-input 

https://github.com/AlaskaAirlines/auro-input/blob/a13630541e129ca3bb1f48bc5ce31bf5abdd74a8/src/base-input.js#L39

### PR DESCRIPTION
**Pull request by blackfalcon**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #75 

**NOTE:** This issue is a pre-requisite to completing https://github.com/AlaskaAirlines/auro-dropdownmenu/issues/37

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

The primary scope of this PR is to address an issue related to the naming g of a slot that is inconsistent with the naming of a similar slot with auro-input. Having similar things in the API named differently will cause confusion with users. Please see the following commits that address this update. 

- fix(helptext): update slot name https://github.com/AlaskaAirlines/auro-dropdown/pull/76/commits/ece890b1011e77bd4d832fee698d16fbec832055
- docs(helptext): update all demo resources for change https://github.com/AlaskaAirlines/auro-dropdown/pull/76/commits/ee0f20861018bf5fbfb17141d30864a4b8f12d32

The following commits were added to update the repo for general support and performance. 

- refactor(docs): remove comment delete function https://github.com/AlaskaAirlines/auro-dropdown/pull/76/commits/adc8eac522de6eeaf6422c8d7cf987e0cde96e02
- docs(examples): add new html file to easily view demo page https://github.com/AlaskaAirlines/auro-dropdown/pull/76/commits/5a72f822d57deedd2a5062265021c9df16e443c6
- chore(deps): update dependencies and vulnerabilities https://github.com/AlaskaAirlines/auro-dropdown/pull/76/commits/30131871f177add9cee33912a1d201e85b080962
- refactor(ie): remove ie support and post-processing of assets https://github.com/AlaskaAirlines/auro-dropdown/pull/76/commits/74f160f30ccb75be5b5d8715a750138635ccb65c

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [x] Other (resolve bug)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: scripts/generateDocs.js [hunk 1]
```diff
@@ -53,12 +53,6 @@ function formatTemplateFileContents(content, destination) {
   result = result.replace(/\[namespace]/g, nameExtractionData.namespace);
   result = result.replace(/\[Namespace]/g, nameExtractionData.namespaceCap);
 
-  /**
-   * Strip all markdown-magic comments
-   */
-  result = result.replace(/<!-- AURO(.*?)-GENERATED-CONTENT(.*?)-->/g, '');
-  result = result.replace(/<!-- The below (.*?) is automatically added from(.*?)-->/g, '');
-
```
Comment (jason-capsule42) [inline scripts/generateDocs.js]: This shouldn't be removed. We put the comment at the top of the readme about the compiled comment and this should be left in place.

File: src/auro-dropdown.js [hunk 2]
```diff
@@ -17,13 +17,13 @@ import Popover from "../lib/popover";
...
  * @attr { Boolean } disabled - If declared, the dropdown is not interactive.
- * @attr { Boolean } error - If declared in combination with `bordered` property or `helperText` slot content, will apply red color to both.
+ * @attr { Boolean } error - If declared in combination with `bordered` property or `helpText` slot content, will apply red color to both.
  * @attr { Boolean } inset - If declared, will apply padding around trigger slot content.
  * @attr { Boolean } rounded - If declared, will apply border-radius to trigger and default slots.
  * @attr { Boolean } toggle - If declared, the trigger will toggle the show/hide state of the dropdown.
  * @slot - Default slot for the popover content.
  * @slot label - Defines the content of the label.
- * @slot helperText - Defines the content of the helperText.
+ * @slot helpText - Defines the content of the helpText.
```
Comment (jason-capsule42) [inline src/auro-dropdown.js:27]: This makes it a breaking change and needs to have the commit comment to go with that.

I'm not really sure why this change needs to happen. This whole PR seems to really revolve around the changing `helperText` to `helpText` which I am unsure justifies implementing a breaking change to the API.

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

File: src/auro-dropdown.js (hunk 2 above)
Comment (blackfalcon) [reply src/auro-dropdown.js:27]: Updated PR to release v2.0

> I'm not really sure why this change needs to happen.

This slot is inconsistent with other input elements that share a similar slot. This was a miss on my part from the initial PR to release this, so this needs to be corrected.

File: scripts/generateDocs.js (hunk 1 above)
Comment (blackfalcon) [reply scripts/generateDocs.js]: This issue is to be discussed in a larger forum.

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**